### PR TITLE
[TEVA-4337] move pagination stats to below pagination controls

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-4337

## After

Screenshot 2022-07-26 at 16 22 15

### CODE REVIEW

File: app/views/vacancies/index.html.slim [hunk 1]
```diff
@@ -44,3 +43,5 @@
         - else
           = render "vacancies/search/no_results"
       = govuk_pagination(pagy: @pagy)
+      - if @vacancies.any?
+        = render "vacancies/search/sort_and_stats", form: @form, vacancies_search: @vacancies_search, vacancies: @vacancies
```
Comment (cesidio) [inline app/views/vacancies/index.html.slim:47]: will the options to sort vacancies appear at the bottom of the page?

Comment (alexbowen) [reply app/views/vacancies/index.html.slim:47]: they shouldnt, i didnt pay enough attention, sort should be top stats bottom, will change

File: app/views/vacancies/search/_sort.html.slim [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.govuk-grid-row
+  .govuk-grid-column-full
```
Comment (cesidio) [inline app/views/vacancies/search/_sort.html.slim:2]: Didn't we say that we should not use `govuk-grid-column-full` when you have something that takes the full row?

File: app/views/vacancies/search/_stats.html.slim [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.govuk-grid-row
+  .govuk-grid-column-full
```
Comment (cesidio) [inline app/views/vacancies/search/_stats.html.slim:2]: Same here, `govuk-grid-column-full` seems redudntant

File: spec/system/jobseekers_can_search_for_jobs_spec.rb [hunk 4]
```diff
@@ -10,7 +10,7 @@
 
     it "displays page 1 jobs" do
       within("ul.search-results") { expect(page).to have_css("li", count: 2) }
-      expect(page).to have_css(".search-results-sorting__stats", text: strip_tags(I18n.t("app.pagy_stats_html", from: 1, to: 2, total: 6, type: "results")))
+      expect(page).to have_css(".govuk-grid-column-full", text: strip_tags(I18n.t("app.pagy_stats_html", from: 1, to: 2, total: 6, type: "results")))
```
Comment (cesidio) [inline spec/system/jobseekers_can_search_for_jobs_spec.rb:13]: Checking that something is inside `govuk-grid-column-full` is not useful as it's not a specific class. I'd just check the standard
`expect(page).to have_content strip_tag....`

**Review by cesidio — APPROVED** [review]

**Pull request merged**